client/payment-details/summary/index.tsx

```diff
@@ -218,13 +232,42 @@ const PaymentDetailsSummary = ( {
 					</div>
 				</div>
 			</CardBody>
-			<CardFooter>
+			<CardDivider />
+			<CardBody>
 				<LoadableBlock isLoading={ isLoading } numLines={ 4 }>
 					<HorizontalList
 						items={ composePaymentSummaryItems( { charge } ) }
 					/>
 				</LoadableBlock>
-			</CardFooter>
+			</CardBody>
+			{ displayCaptureAuthorization &&
+				authorization &&
+				! authorization.captured && (
+					<Loadable isLoading={ isLoading } placeholder="">
+						<CardFooter className="payment-details-capture-notice">
+							<div className="payment-details-capture-notice__section">
+								<div className="payment-details-capture-notice__text">
+									{ `${ __(
+										'You need to capture this charge before',
+										'woocommerce-payments'
+									) } ` }
+									<b>{ authorization.capture_by }</b>
+								</div>
+								<div className="payment-details-capture-notice__button">
+									<CaptureAuthorizationButton
+										orderId={ charge.order?.number || 0 }
+										paymentIntentId={
+											charge.payment_intent || ''
+										}
+										buttonIsPrimary={ true }
+										buttonIsSmall={ false }
+										paymentIsCaptured={ isCaptured }
```

Comment:
Given one of the rendering conditions is `! authorization.captured`, I'd expect that we could drop paymentIsCaptured right away. But I probably miss the mocking factor, can you elaborate, please?